DataFiles/unit-mapping.csv

```diff
@@ -59,8 +62,10 @@
 "mm/(m⋅K)", "mm/(m·K)"
 "mNHN", "?"
 "MN/m2", "MN/m²"
+"m/Nm²", "m/(N·m²)"
 "MN/m3", "MN/m³"
-"(m².K)/W", "(m²·K)/W"
+"(m².K)/W", "m²·K/W"
+"(m²·K)/W", "m²·K/W"
```

Comment:
duplicate

Reply:
I now see they are not :)

Reply:
Yep :-). That's why I added validation of this file before uploading it to bSDD, to prevent duplicates